screens/HomeAuth.js

```diff
@@ -23,17 +24,26 @@ class HomeAuth extends Component {
   };
 
   state = {
-    loading: false
+    fbLoading: false,
```

Comment:
Do we only need the two different loading bools because we need to show the loading animation per button? It might be better to have a single "tokenLoading" bool, and also track the platform you are calling for as it's own state. Then we can add more auth methods and not have to add more state. 

It also forces the state of these bools to be mutually exclusive, which is the behavior we want.

Reply:
Thanks Bassam! I think we probably still want to have a "loadingPlatform" state field so that we can still show the correct <Button> loading state. The new "isLoading" check would be to check that the state.loading bool is true, and also that the "loadingPlatform" matches "my" button (so if it were "facebook", then the "facebook" button would know it should display as loading, and same for "google").

Reply:
just added `loadingPlatform`, please let me know if this is what you had in mind